autoware_launch/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/behavior_velocity_planner.param.yaml

```
@@ -10,6 +10,7 @@
     launch_occlusion_spot: true
     launch_no_stopping_area: true
     launch_run_out: false
+    launch_speed_bump: true
```

```suggestion
    launch_speed_bump: false
```
@mehmetdogru Is it ok to make the default false? People that need it can enable it.

---

Sure @xmfcx